src/api/app/helpers/webui/staging/workflow_helper.rb

```diff
@@ -23,10 +23,11 @@ def review_progress(staging_project)
   end
 
   def testing_progress(staging_project)
-    notdone = staging_project.checks.pending.size
     allchecks = staging_project.checks.size + staging_project.missing_checks.size
+    return 0 if allchecks == 0
+
+    notdone = staging_project.checks.pending.size
 
-    return 100 if allchecks == 0
```

Comment:
What's going happen if `notdone =  0`?

Reply:
It will be returned 0.

Reply:
I would rather think 100 - 0, aka 100. And this is exactly what the bug is all about.

In the situation described in the ticket, there are 0 checks and 1 missing check. That means allchecks = 1, notdone = 0.

Reply:
as @coolo said, If I'm not wrong, it will be like:

```100 - notdone * 100 / allchecks``` is like

```100 - ((notdone * 100) / allchecks)```

which will be in the end ```100 - 0 / allchecks```, no?

Reply:
You are both right. :man_facepalming: I will fix this.

Reply:
¡Cálculo de puntos antes del cálculo de líneas! :smile: